academic-calendar/react/src/components/CareerFilter.jsx

```diff
@@ -7,19 +7,18 @@ import API from '../api'
 const DropDownItems = (props) => {
   return Object.entries(props.careers).map((entry, index) => {
     return (
-      <Dropdown.Item key={index} eventKey={entry[1].code} active={entry[1].code === props.career}>{entry[1].description}</Dropdown.Item>
+      <Dropdown.Item data-testid="career-option" key={index} eventKey={entry[1].code} active={entry[1].code === props.career}>{entry[1].description}</Dropdown.Item>
```

Comment:
Team: review https://testing-library.com/docs/queries/bytestid/ and determine if polluting the code with data-testid is the best and/or only option here

Reply:
rewrote the test to query by text instead of testid